src/odemis/driver/hamamatsurx.py

```diff
@@ -672,6 +677,34 @@ def _setPLLMode(self, value: bool):
 
         return value
 
+    def GetShutter(self) -> bool:
+        """
+        Get the current value from the trigger delay HW (RemoteEx: delay A).
+        :return: (bool) current state of the shutter
+        """
+        shutter_raw = self.parent.DevParamGet(self.location, "Shutter")  # returns a list
+        if shutter_raw[0] == "Open":
+            shutter = True
+            self.state._set_value(model.ST_RUNNING, force_write=True)
+        elif shutter_raw[0] == "Closed":
+            shutter = False
+        else:
+            logging.warning("Unexpected streak mode %s", shutter_raw)
+            shutter = False  # safer!
+
+        return shutter
+
+    def _setShutter(self, value: bool) -> bool:
+        """
+        Updates the shutter state VA.
+        :param value: value to be set
+        :return: (bool) shutter state
+        """
+        self.parent.DevParamSet(self.location, "Shutter", "Open" if value else "Closed")
```

Comment:
Typically it's best for the rest of Odemis to be certain that the shutter is actually closed/opened by the time this setter function returns.
For the acquisition to work reliably, we need to make sure the shutter is actually opened otherwise, Odemis might start acquisition while the shutter is opening. That said, typically the exposure time is quite long (> 1s), and the shutter opening time is most likely < 0.1s, so that might have very little effect. 

In the other direction, it's safer to make sure the shutter is closed, before doing "something" else, which might involve a lot of light.
So, please have a look at whether the function returns after the move is complete, or you should add a small `sleep()`.